src/sessions/softphone-session-handler.ts

```diff
@@ -19,24 +20,22 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
     }
   }
 
-  async handleSessionInit (session: any): Promise<void> {
+  async handleSessionInit (session: IJingleSession): Promise<void> {
     await super.handleSessionInit(session);
     if (this.sdk._config.autoConnectSessions) {
       return this.acceptSession(session, { id: session.id });
     }
-
-    // todo: will need to update this to call streamingClient.rtcSessionAccepted
   }
 
-  async acceptSession (session: any, params: IAcceptSessionRequest): Promise<any> {
+  async acceptSession (session: IJingleSession, params: IAcceptSessionRequest): Promise<any> {
     let stream = params.mediaStream || this.sdk._config.defaultAudioStream;
     if (!stream) {
       this.log(LogLevels.debug, 'No mediaStream provided, starting media');
-      stream = await startAudioMedia();
+      stream = await startMedia({ audio: true });
       this.log(LogLevels.debug, 'Media start');
     }
     this.log(LogLevels.debug, 'Adding media to session');
-    this.addMediaToSession(session, stream);
+    await this.addMediaToSession(session, stream);
     session._outboundStream = stream;
```

Comment:
yes, that was intentional because in the future we will probably support screenshare and video on the same session at the same time. This was for future proofing.